Android.mk

```diff
@@ -135,6 +135,9 @@ ifeq ($(KERNELFLINGER_USE_NVME_RPMB),true)
     KERNELFLINGER_CFLAGS += -DNVME_RPMB
 endif
 
+ifeq ($(TARGET_USE_TPM),true)
+    KERNELFLINGER_CFLAGS += -DSECURE_STORAGE_TPM
```

Comment:
what this difference between "SECURE_STORAGE_TPM" and "USE_TPM"?  Why we need both?

Reply:
Will not use it.
SECURE_STORAGE_TPM will be a enum item, not a macro define now.